store/schedule_test.go

```diff
@@ -203,35 +328,77 @@ func TestValidateSchedule(t *testing.T) {
 			CronExpression: "*/5 * * * *",
 		}
 
-		errs := s.ValidateSchedule()
+		errs := s.ValidateSchedule(a, conf)
 		if len(errs) == 0 {
 			t.Fatal("expected errors, got none")
 		}
 	})
 
 	t.Run("valid recurring schedule", func(t *testing.T) {
+		conf := conf2.AppLevelConfiguration{
+			FutureScheduleCreationPeriod: 7,
+			FiredScheduleRetentionPeriod: 1,
+			PayloadSize:                  1024,
+			HttpRetries:                  1,
+			HttpTimeout:                  1000,
+		}
+
+		a := App{
+			AppId:      "appId",
+			Partitions: 0,
+			Active:     false,
+			Configuration: Configuration{
+				FutureScheduleCreationPeriod: 7,
+				FiredScheduleRetentionPeriod: 1,
+				PayloadSize:                  1024,
+				HttpRetries:                  1,
+				HttpTimeout:                  1000,
+			},
+		}
+
 		s := &Schedule{
```

Comment:
rename this variable

Reply:
Brevity in variable names wherever the context is understood is preferred. Please check this https://go.dev/wiki/CodeReviewComments#variable-names